src/pages/aclmanagement.html

```diff
@@ -654,7 +678,11 @@ <h1 id="acl-management-view-title">
                                         <div class="small" ng-if="ruleData.role.$touched && !ruleData.role.$valid">
                                             <small>{{'acl_management.errors.role_length_too_short' | translate}}</small>
                                         </div>
+                                        <div class="small float-lg-left prefix-warning-text" ng-if="rule.role.startsWith(doublePrefix) && !showPrefixWarningIcon">
```

Comment:
I think these checks needs to be reconsidered, because they look quite strange at the moment. How can a role starts with a CUSTOM_CUSTOM_ and not with a CUSTOM_ before that which is the meaning of the showPrefixWarningIcon if I read it properly?